pkg/api/fs.go

```diff
@@ -164,14 +165,26 @@ func GetProperLines(regex string, fileContent string) ([]string, error) {
 }
 
 // Apply
-// 1st argument is tuple of rules, where 1st element is ruleset identifier and 2nd is Name
+// 1st argument is tuple of rules, where 1st element is ruleset identifier, 2nd is Name and 3rd is `IsString` which takes true or false
 func (f *FsApi) Apply(rulesHistory [][]string) (int, error) {
```

Comment:
Don't you think it would be better to create structure or interface for this?

Reply:
Definitely